src/main/java/teammates/common/datatransfer/FeedbackResponseAttributes.java

```diff
@@ -77,6 +84,8 @@ public FeedbackResponseAttributes(FeedbackResponseAttributes copy) {
         this.recipientEmail = copy.recipientEmail;
         this.recipientSection = copy.recipientSection;
         this.responseMetaData = copy.responseMetaData;
+        this.createdAt = copy.getCreatedAt();
+        this.updatedAt = copy.getUpdatedAt();
```

Comment:
Make these `this.*At = copy.*At` instead. We want the copy to be a perfect copy, i.e if the source has null field, we don't need the copy to be initialized with default value.